test/unit/math/mix/scal/prob/normal_cdf_log_test.cpp

```diff
@@ -0,0 +1,18 @@
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
+
+TEST(mathMixScalFun, lcdf_derivatives) {
+  auto f = [](const double mu, const double sigma) {
+    return [=](const auto& y) { return stan::math::normal_lcdf(y, mu, sigma); };
+  };
+  stan::test::expect_ad(f(0.0, 1.0), -50.0);
+  stan::test::expect_ad(f(0.0, 1.0), -20.0 * stan::math::SQRT_2);
+  stan::test::expect_ad(f(0.0, 1.0), -5.5);
+  stan::test::expect_ad(f(0.0, 1.0), 0.0);
+  stan::test::expect_ad(f(0.0, 1.0), 0.15);
+  stan::test::expect_ad(f(0.0, 1.0), 1.14);
+  stan::test::expect_ad(f(0.0, 1.0), 3.00);
+  stan::test::expect_ad(f(0.0, 1.0), 10.00);
+  stan::test::expect_ad(f(-1.0, 2.0), 1.50);
+  stan::test::expect_ad(f(2.0, 1.0), 0.50);
```

Comment:
Can you add the failing test with Bob's tolerance stuff in here:

```
stan::test::ad_tolerances tols;
tols.grad_hessian_grad_hessian_ = 1e1;

stan::test::expect_ad(tols, f(0.0, 1.0), 0.1 * stan::math::SQRT_2);
```

There are a fair number of things that fail with third order autodiff and that's okay. Maybe we come back and fix them sometime, but having these tests around I think will be good for bookkeeping (if we need to find these functions we can grep for them).

Reply:
No problem. I've added this